scss/bootstrap/mixins/_checkbox-switch.scss

```diff
@@ -0,0 +1,34 @@
+@mixin checkbox-switch-context($colour) {
```

Comment:
Please document the mixins: http://sassdoc.com/annotations/, https://pineco.de/document-sass-projects-sassdoc/ I know we are barely doing it anywhere, but let's start from somewhere